.github/workflows/code.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - 'master'
+      - 'spec-test-*'
+
+jobs:
+  sync-vagrant:
+    if: github.repository == 'hashicorp/vagrant-blackbox'
```

Comment:
This section can be removed. We should only be syncing from vagrant and never to vagrant.